packages/components/src/styles/colors.ts

```diff
@@ -33,11 +46,16 @@ const themeLight = {
   primaryColor: '#010205',
   primaryColorDarken: '#000000',
   backgroundColor: '#FFF',
+  backgroundColorDarken: '#e0e0e0',
```

Comment:
I am wondering if all those new colors really belong into the theme or into the tag utils. Reusability is not given and I have the feeling this increases the chances to use the wrong colors anywhere else.

Reply:
@denyo I added "TODO" comment into `generateThemesObject.ts`:
`
    // TODO: since such colors are being used only in Tag, it makes sense to move such colors into tag utils, in order to decrease the chances to use the wrong colors
`

I hope this is what you meant by "mark as todo".